projects/step-frontend/src/lib/modules/_common/components/sidebar/sidebar.component.html

```diff
@@ -66,6 +71,11 @@
           <span class="sidebar-aligned-text">Home</span>
         </label>
       </div>
+      <div *ngIf="!(isOpened$ | async)" class="sidebar-collapsable-tab" (mouseup)="navigateTo('home', $event)">
+        <label>
+          <step-icon class="sidebar-home-and-bookmark-icon" name="star" (click)="addBookmark()"></step-icon>
+        </label>
+      </div>
```

Comment:
```suggestion
      <div *ngIf="!(isOpened$ | async)" class="sidebar-collapsable-tab" (click)="addBookmark()">
        <label>
          <step-icon class="sidebar-home-and-bookmark-icon" name="star"></step-icon>
        </label>
      </div>
```

Also make sense to optimize the template a bit. As both home and bookmark menu items are displayed one by another, they can be grouped in one container with single check of isOpenened flag
```
<ng-container *ngIf="!(isOpened | async)">
    <div class="sidebar-collapsable-tab" (mouseup)="navigateTo('home', $event)">
         <label>
          <step-icon class="visible-when-collapsed" name="home"></step-icon>
          <span class="sidebar-aligned-text">Home</span>
        </label>
      </div>
    <div class="sidebar-collapsable-tab" (click)="addBookmark()">
        <label>
          <step-icon class="sidebar-home-and-bookmark-icon" name="star"></step-icon>
        </label>
      </div>
</ng-container>
```